test/e2e-tests.sh

```diff
@@ -120,6 +120,8 @@ kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-leader-election --type=m
 add_trap "kubectl get cm config-leader-election -n ${SYSTEM_NAMESPACE} -oyaml | sed '/.*enabledComponents.*/d' | kubectl replace -f -" SIGKILL SIGTERM SIGQUIT
```

Comment:
Does this trap also has to delete pods to move them into _non HA_ mode back?

Reply:
Yes. But we don't scale down the deployments at the end of the script. We need to scale down to 1 before restarting the pod otherwise we'll have two pods without leader election clashing with each other. Do we want to do that? This is the last test in the script

Reply:
Sure, let's do both. Basically I'd like to return to the exact situation as before the test started.

Reply:
OK.